f5_openstack_agent/lbaasv2/drivers/bigip/agent.py

```diff
@@ -71,7 +71,7 @@ def main():
     """F5 LBaaS agent for OpenStack."""
     cfg.CONF.register_opts(OPTS)
     cfg.CONF.register_opts(manager.OPTS)
-    cfg.CONF.register_opts(interface.OPTS)
+    cfg.CONF.register_opts(INTERFACE_OPTS)
```

Comment:
this way seems to make it quite complicated. should probably Try this way: copy the content of  INTERFACE_OPTS  into this file , then  this line can be always the same between each branch.  each branch has the same and we dont have to consider queens differently.

Reply:
in that way, we do not need a separate queens branch.   we can still use our existing 9.8 branch for queens, as well as the original M N, O. thus,  avoid many inconveniences for maintenance work.

Reply:
agree